observatory_platform/tests/test_dataset_api.py

```diff
@@ -89,7 +89,7 @@ def test_get_dataset_releases(self):
 
             # Get releases
             actual = api.get_dataset_releases(dag_id=dag_id, entity_id=entity_id)
-            self.assertListEqual(expected, actual)
+            self.assertListEqual([e.to_dict() for e in expected], [a.to_dict() for a in actual])
```

Comment:
The actual object should be compared (it is compared with the "__eq__" function), as the dict form is different.